FirebaseDynamicLinks/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 10.0.1
```

Comment:
Note: 10.1.0 is planned for release today and I'm not aware of any plans for a 10.0.1 release. I think if you do `# unreleased` it should automatically get changed to `10.2.0` or a `10.1.x` point-release, whichever comes first, by our release tooling.

Reply:
Thanks for the comment. I added `#unreleased`